### PR TITLE
Add systemd unit for timeoverflow service and crontab entry for scheduled job

### PR DESCRIPTION
**Pull request by enricostano**
- [x] Make `ruby` available running systemd unit

### CODE REVIEW

File: roles/webserver/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- name: Create crontab entries for scheduled job
+  cron:
+    name: send_recent_posts_to_online_members
+    user: timeoverflow
+    weekday: 1
+    hour: 9
+    minute: 0
+    state: present
+    job: >
+      /bin/bash -l -c 'cd /var/www/timeoverflow/current && bin/rails runner -e production '\''OrganizationNotifierService.new(Organization.all).send_recent_posts_to_online_members'\'' >> log/cron_log.log 2>&1'
```
Comment (sauloperez) [inline roles/webserver/tasks/main.yml:16]: My God, hasn't that `Organization.all` blown up our DB already??

**Review by sauloperez — CHANGES_REQUESTED** [review]

File: roles/webserver/tasks/main.yml (hunk 1 above)
Comment (enricostano) [reply roles/webserver/tasks/main.yml:16]: I would leave as it is right now (copied from production) and improve it later, feel free to open a new issue. 🙏

File: roles/webserver/templates/unicorn_service.j2 [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+[Service]
+Type=simple
+SyslogIdentifier=timeoverflow-unicorn
+User=timeoverflow
+PIDFile=/var/www/timeoverflow/shared/tmp/pids/unicorn.pid
+WorkingDirectory=/var/www/timeoverflow/current
+EnvironmentFile=-/etc/default/timeoverflow
+
+ExecStart=/var/www/timeoverflow/current/bin/unicorn_rails
```
Comment (enricostano) [inline roles/webserver/templates/unicorn_service.j2]: https://gist.github.com/arteezy/5d53d99f6ee617fae1f0db0576fdd418

File: roles/webserver/tasks/main.yml (hunk 1 above)
Comment (sauloperez) [reply roles/webserver/tasks/main.yml:16]: Done in https://github.com/coopdevs/timeoverflow-provisioning/issues/24

**Review by sauloperez — APPROVED** [review]

**Review by danypr92 — APPROVED** [review]

**Pull request merged**